app/controllers/provider/sessions_controller.rb

```diff
@@ -87,12 +87,12 @@ def auth_params
   end
 
   def sso_params
-    params.permit(:token, :expires_at, :redirect_url, :system_name, :code).merge(request: request)
+    params.permit(%i[token expires_at redirect_url system_name code]).to_h.merge(request: request)
```

Comment:
Is the `to_h` because of a warning with security on user provided params?
Otherwise it is not needed

Reply:
@hallelujah Changes done